### PR TITLE
Controllers specs

### PR DESCRIPTION
**Pull request by Marcraphael12**
### Project requirements
- Add RSpec as a gem to your project.
- Create a Request spec file for all your controllers.
- Make sure that for each action you check:
    - If response status was correct.
    - If a correct template was rendered.
    - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  let(:user) do
+    User.create(
+      name: 'Marc',
+      photo: 'http://www.image.com',
+      bio: 'Developer',
+      post_counter: 5
+    )
```
Comment (iambenkis) [inline spec/requests/posts_spec.rb:10]: - [x] Keep doing the great work 👍

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @Marcraphael12 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  Routes are included
✔️ Controllers specs are passing 

## Suggested changes

- [ ] Kindly fix the broken link attached to the logo image , it will keep your README professinal 😄 
Capture1


- [ ] I will also suggest you to personalize the MIT license because  To make this MIT license your own, it's important to replace the placeholder [YOUR NAME] with your actual name, and [APP TYPE] with the specific type of application you're working on.

Capture


_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: spec/requests/posts_spec.rb (hunk 1 above)
Comment (Marcraphael12) [reply spec/requests/posts_spec.rb:10]: Thanks 🙂

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @Marcraphael12 

Good job implementing the suggested changes💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Broken link is fixed
-  ✅ License is updated 
-  ✅ Your README looks good
-  ✅ Linters are working properly

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**